core/ir/decode_shared.c

```diff
@@ -174,6 +174,33 @@ const char *const size_names[] = {
     "OPSZ_eighth_16_vex32_evex64",
 };
 
+/* AArch64 Scalable Vector Extension's vector length in bits. */
+int sve_veclen;
+int sve_veclens[] = { 128,  256,  384,  512,  640,  768,  896,  1024,
+                      1152, 1280, 1408, 1536, 1664, 1792, 1920, 2048 };
+
+void
+dr_set_sve_vl(int vl)
+{
+    /* i3044 TODO: Vector length will be read from h/w when running on SVE.
+     * CLIENT_ASSERT(!proc_has_feature(FEATURE_SVE),
```

Comment:
Is this CLIENT_ASSERT commented-out because we're hitting it now?

Reply:
No, it's what will _probably_ be required as part of upcoming patches to enable execution on SVE h/w.
As it's commented out code, I'll delete to avoid confusion.